src/pages/ProfileRegisterNormal.tsx

```diff
@@ -74,12 +95,12 @@ export default function ProfileRegisterNormal() {
           <p className="lg:text-[1.6rem] md:text-[1.2rem] sm:text-[1.2rem] font-normal text-gray-400 mb-[3.2rem]">
             *내가 사는 지역은 언제든 수정 가능해요!
           </p>
-          <ProfileChips regions={regions} selectedRegion={selectedRegion} setSelectedRegion={setSelectedRegion} />
+          <ProfileChips regions={regions} selectedRegions={selectedRegions} setSelectedRegions={setSelectedRegions} />
         </div>
       </div>
       <ButtonWrapper id="profile-register-normal" onClick={handleImgSubmit}>
         <ButtonWrapper.Button
-          disabled={!(selectedImg && selectedMovingType && selectedMovingType)}
+          disabled={selectedMovingType.length === 0 || selectedRegions.length === 0}
```

Comment:
이것보다는
```suggestion
          disabled={!selectedMovingType.length || !selectedRegions.length}
```
이렇게 사용하시면 더 깔끔하지 않을까요??